src/styles/hero.scss

```diff
@@ -0,0 +1,11 @@
+.hero-logo-lockup {
+  width: 400px;
```

Comment:
For smaller screens it's probably best to make this a `max-width` for specificity I think you'll have to change the selector to `.grav-c-hero .grav-c-hero__container .hero-logo-lockup`